funding/batch.go

```diff
@@ -477,6 +513,21 @@ func (b *Batcher) waitForUpdate(channel *batchChannel, firstUpdate bool) error {
 
 	case <-b.cfg.Quit:
 		return errShuttingDown
+
+	case <-deadlineCtx.Done():
+		// If we haven't received any updates within the timeout period,
+		// we'll return an error indicating the peer that failed to
+		// respond in time.
+		log.Infof("Cancelled batch channel opening for peer %x due to "+
+			"timeout. The peer didn't provide any funding updates"+
+			" within the timeout period.",
+			channel.fundingReq.TargetPubkey.SerializeCompressed())
```

Comment:
nit: extract `channel.fundingReq.TargetPubkey.SerializeCompressed()` into a variable at the top as it's being used three times within this method?